lib/htmlValidator.js

```diff
@@ -53,9 +52,24 @@ module.exports = function(app, callback) {
           }
         });
 
+        validatorProcess.stderr.on('data', (data) => {
+          if (`${data}`.includes('No Java runtime present, requesting install.')) {
+            console.error('There was an error in authentication. Please make sure to have Java installed.');
```

Comment:
Let's change the wording here to "No Java runtime detected. Check to make sure Java is installed and in your path."

Both this error and the other one down there can probably use the same language.